### ISSUE TITLE
Austen Accessibility Audit

### ISSUE THREAD

**Issue by nichgray** [CDRH/austen#69]
# Version of site audited

**URL:** https://austen.unl.edu/
**branch:** `a11y_fixes`

# Checklist

  - [ ] Generate lists for this issue using the [Checklist of Products for ADA Title II](https://uofnebraska.sharepoint.com/:x:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project/Planning%20Documents/Checklist%20of%20Products%20for%20ADA%20Title%20II%20Project.xlsx?d=w52aafefaa1a841b09bae916225c22cbb&csf=1&web=1&e=mvYOdi) spreadsheet
  - [ ] Identify sections and URLs that duplicate search/browse for each section
  - [ ] Identify representative items/pages from each section
  - [ ] Identify any other unique, non-item pages within each section
  - [ ] Compile in spreadsheet (linked above)

## `robots.txt`
  - [ ] Upload `robots.txt`. Include:
    - /browse/
    - /search
    - for each section: search/browse/other views with paginated results

### Contents of `robots.txt` for this site:

```
User-agent: *
Disallow: /browse/
Disallow: /search
```

## Site crawl

- [ ] Scan site (dev site if possible) using [WAE tool](https://wae.unl.edu/)

```

```

## Page scans

Pages scanned individually using axeDevTools web browser extension:

## Manual testing

Pages tested manually for things that automated scans cannot detect:

  - [x] Home page
  - [x] Primary "static" pages
    - [ ] 

## Delete robots.txt

- [ ] Remove robots.txt from server
